### PR TITLE
Updated _form.html.erb

### PR DESCRIPTION
**Pull request by joshuaretallick**
Fixes https://github.com/publiclab/plots2/issues/9129

First time doing this, hope I did it correctly!

### CODE REVIEW

**Review by RuthNjeri — APPROVED** [review]
This looks great @joshuaretallick! I have added the issue number which this PR is fixing in the PR description(it is usually part of creating the PR, that way, reviewers have an idea of what you are fixing or what your PR relates to)

I have also restarted the test build since one was failing... it should be okay to merge once all tests pass.

**Comment by joshuaretallick** [comment]
Hi, I'm a little confused. What am I suppose to do now? Submit another pull request? 
Thanks
Josh

**Comment by RuthNjeri** [comment]
Hi Josh, not really...the failing tests are intermittent... I will just keep reopening/closing the PR until it passes

**Comment by joshuaretallick** [comment]
Ok thanks!

**Comment by RuthNjeri** [comment]
@joshuaretallick are you familiar with rebasing? Would you be able to rebase this branch against the recent changes in the upstream main branch? The tests keep on failing more than expected...

If you are unfamiliar with rebasing it's alright I can guide you through it over the weekend or on Monday

**Comment by joshuaretallick** [comment]
Hi Ruth, I haven't done rebasing before. I am happy to look in to it though if you think it is straight fwd enough to try on my own?

**Comment by joshuaretallick** [comment]
Something like this?

git fetch origin           
git rebase origin/master

**Comment by RuthNjeri** [comment]
> Something like this?
> 
> git fetch origin
> git rebase origin/master

Hi @joshuaretallick yes something like that... I usually do:

1. git checkout main (To navigate to the Main branch)
2. git pull upstream main (To pull the latest changes from the Upstream(the Public Lab repo Plots2) to the local Main branch)
3. git checkout joshuaretallick(To navigate back to this branch..that the PR was created from)
4. git rebase main (To rebase the changes from Main to the joshuaretallick branch)
5. If you get any conflicts, follow the steps written [here](https://gist.github.com/RuthNjeri/82f3adeac98c2343a0d7e832da3993b5) from number 2...Otherwise
6. git push --force-with-lease joshuaretallick to push the changes to this PR without having to write up a new commit.

Let me know if it makes sense...or if you have any questions...
Also, just a side note for the next time you create a branch, it is recommended to name your branch according to what you are working on...instead of ` joshuaretallick` you can name it `update-form`...you don't need to change the branch name now...this is just for future purposes.

**Comment by RuthNjeri** [comment]
Hi @joshuaretallick, were you able to do a rebase?

**Comment by RuthNjeri** [comment]
Hi @joshuaretallick,  let me know if you are still able to work on this issue, I can reassign it if you are not able to...

**Comment by joshuaretallick** [comment]
Apologies for the delay here Ruth, I have been assigned on to another project which has been a bit time consuming.
I tried rebasing but not sure it worked.
Could you please send me some instructions so I do it properly?
No problem if you need to reassign this
Thanks
J

Josh Retallick
Director
[Old Empire]<http://www.old-empire.co.uk/>
Phone:  +44 7826 855 848
Email:  josh@old-empire.co.uk<mailto:josh@old-empire.co.uk>
[Facebook]<https://www.facebook.com/oldempirepro>[Instagram]<https://www.instagram.com/oldempire/>[Website]<http://www.old-empire.co.uk/>






On 16 Feb 2021, at 20:35, Ruth <notifications@github.com<mailto:notifications@github.com>> wrote:



Hi @joshuaretallick<https://github.com/joshuaretallick>, let me know if you are still able to work on this issue, I can reassign it if you are not able to...

**Comment by RuthNjeri** [comment]
Hi @joshuaretallick, that's okay, I sent some steps to follow [here](https://github.com/publiclab/plots2/pull/9133#issuecomment-775172012) have you managed to follow them?

**Comment by joshuaretallick** [comment]
Hi Ruth,
I think this is done now, sorry I have been such a pain!

Josh Retallick
Director
[Old Empire]<http://www.old-empire.co.uk/>
Phone:  +44 7826 855 848
Email:  josh@old-empire.co.uk<mailto:josh@old-empire.co.uk>
[Facebook]<https://www.facebook.com/oldempirepro>[Instagram]<https://www.instagram.com/oldempire/>[Website]<http://www.old-empire.co.uk/>






On 18 Feb 2021, at 15:06, Ruth <notifications@github.com<mailto:notifications@github.com>> wrote:



Hi @joshuaretallick<https://github.com/joshuaretallick>, that's okay, I sent some steps to follow here<https://github.com/publiclab/plots2/pull/9133#issuecomment-775172012> have you managed to follow them?

File: app/views/locations/_form.html.erb [hunk 1]
```diff
@@ -192,11 +192,12 @@
...
     if (event.errors.length > 0 && event.errors.includes("Error: tag already exists.")) {
-        notyNotification('relax', 3000, 'error', 'topCenter', 'Location Already Saved.');
+        notyNotification('relax', 3000, 'error', 'topRight', 'Location Already Saved.');
       } else if (event.saved.length > 0) {
-        notyNotification('relax', 3000, 'success', 'topCenter', 'Location Saved');
+        notyNotification('relax', 3000, 'success', 'topRight', 'Location Saved');
       } else {
-        notyNotification('relax', 3000, 'error', 'topCenter', 'Something went wrong. Contact administrator.');
+        notyNotification('relax', 3000, 'error', 'topRight', 'Something went wrong. Contact administrator.');
+      }
```
Comment (RuthNjeri) [inline app/views/locations/_form.html.erb:200]:
```suggestion
```
Hi @joshuaretallick I think line 200 is the issue...you have added an extra bracket `}` and this is making the map not get loaded and thus a test failing... sorry I am catching this late this PR should be merged after you fix this...

**Review by RuthNjeri — CHANGES_REQUESTED** [review]

File: app/views/locations/_form.html.erb (hunk 1 above)
Comment (RuthNjeri) [reply app/views/locations/_form.html.erb:200]: Oh and you haven't been a pain...we just discovered the real cause of the issue now 😅 ...debugging!

**Comment by joshuaretallick** [comment]
Ok hopefully this time it works!

Josh Retallick
Director
[Old Empire]<http://www.old-empire.co.uk/>
Phone:  +44 7826 855 848
Email:  josh@old-empire.co.uk<mailto:josh@old-empire.co.uk>
[Facebook]<https://www.facebook.com/oldempirepro>[Instagram]<https://www.instagram.com/oldempire/>[Website]<http://www.old-empire.co.uk/>






On 22 Feb 2021, at 17:34, Ruth <notifications@github.com<mailto:notifications@github.com>> wrote:



@RuthNjeri commented on this pull request.

________________________________

In app/views/locations/_form.html.erb<https://github.com/publiclab/plots2/pull/9133#discussion_r580444398>:

>        } else {
-        notyNotification('relax', 3000, 'error', 'topCenter', 'Something went wrong. Contact administrator.');
+        notyNotification('relax', 3000, 'error', 'topRight', 'Something went wrong. Contact administrator.');
+      }


Oh and you haven't been a pain...we just discovered the real cause of the issue now 😅 ...debugging!

**Comment by RuthNjeri** [comment]
Hi @joshuaretallick have you managed to work on this? It only requires a small update then we can finally merge this

**Comment by joshuaretallick** [comment]
Hi Ruth,
I sorted this directly on Github, it looks like the build is still failing
Josh Retallick
Director
[Old Empire]<http://www.old-empire.co.uk/>
Phone:  +44 7826 855 848
Email:  josh@old-empire.co.uk<mailto:josh@old-empire.co.uk>
[Facebook]<https://www.facebook.com/oldempirepro>[Instagram]<https://www.instagram.com/oldempire/>[Website]<http://www.old-empire.co.uk/>






On 25 Feb 2021, at 07:37, Ruth <notifications@github.com<mailto:notifications@github.com>> wrote:



Hi @joshuaretallick<https://github.com/joshuaretallick> have you managed to work on this? It only requires a small update then we can finally merge this

**Comment by RuthNjeri** [comment]
It doesn't seem as though the changes were committed. Because the bracket was still present. I have committed the suggested change I made...let's see if the build passes.

**Review by RuthNjeri — APPROVED** [review]
It looks good now. Thanks @joshuaretallick

**Review by RuthNjeri — APPROVED** [review]

**Comment by joshuaretallick** [comment]
Thanks Ruth, sorry for all the messing around

Josh Retallick
Director
[Old Empire]<http://www.old-empire.co.uk/>
Phone:  +44 7826 855 848
Email:  josh@old-empire.co.uk<mailto:josh@old-empire.co.uk>
[Facebook]<https://www.facebook.com/oldempirepro>[Instagram]<https://www.instagram.com/oldempire/>[Website]<http://www.old-empire.co.uk/>






On 25 Feb 2021, at 09:42, Ruth <notifications@github.com<mailto:notifications@github.com>> wrote:



@RuthNjeri approved this pull request.

File: app/views/locations/_form.html.erb (hunk 1 above)
Comment (cesswairimu) [reply app/views/locations/_form.html.erb]: Great catch Ruth thanks :tada:

**Comment by cesswairimu** [comment]
Thanks @joshuaretallick  for sticking with this..amd thanks a bunch @RuthNjeri  for the reviews :tada: 
Merging now

**Pull request merged**